packages/application-shell/package.json

```diff
@@ -82,8 +93,8 @@
     "is-retina": "1.0.3",
     "lodash": "4.17.20",
     "memoize-one": "5.1.1",
-    "moment": "^2.24.0",
-    "moment-timezone": "^0.5.26",
+    "moment": "2.29.1",
+    "moment-timezone": "0.5.32",
```

Comment:
Any particular reason for removing the range?

Reply:
I had quite big trouble when updating deps cause `@types` had ranges and resolved to different versions has a result. Those I decided to pin. We can unpin moment if you want. Do you remember what the reson was not to pin it?

Reply:
The only reason was to be able to use newer versions of moment from the consumer side, for example like we do with uikit etc.

If there are weird issues we can keep it pinned, we just need to be aware of it.